direct_db_manager.py

```
@@ -12,7 +15,11 @@
 import sqlite3
 import json
 import hashlib
+from PyQt5.QtCore import QThread, pyqtSignal
 import resources
+import queue
+import weakref
```

Ruff (0.12.2)

`weakref` imported but unused

Remove unused import: `weakref`

(F401)